test/factories/file_submissions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
All of the other Factories are named singular, this filename is plural. Rails can be very picky about pluralization (I'm not sure if it is in this case but it would be good to correct this just in case).